lib/virtfs/ext3/file.rb

```diff
@@ -0,0 +1,26 @@
+module VirtFS::Ext3
+  class File
+    def initialize(dir_entry, superblock)
+      @bs = superblock
+      @de = dir_entry
+    end
+
+    def to_h
+      { :directory? => @de.dir?,
+        :file?      => @de.file?,
+        :symlink?   => @de.symlink? }
+    end
+
+    def fs
+      @de.fs
+    end
+
+    def size
+      @de.size
+    end
+
+    def close
+      @de.close
+    end
+  end # class File
```

Comment:
@movitto The file class also requires a `raw_read` method (I don't understand how read tests can pass without this).
it probably should also implement `close` even if it's a no-op.
In addition, if available, the `atime`, `ctime` and `mtime` methods should be implemented.